### PR TITLE
MTA-2196: Release Notes for MTA 7.0.2

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTA-2196](https://issues.redhat.com/browse/MTA-2196)
    * [MTA-2263](https://issues.redhat.com/browse/MTA-2263)

### REVIEW

* [MTA 7-0-2](https://deploy-preview-862--windup-documentation.netlify.app/docs/release-notes/master/#mta-7-0-2)

   *  [New features](https://deploy-preview-862--windup-documentation.netlify.app/docs/release-notes/master/#rn-new-features-7-0-2_release-notes)
    * [Known issues](https://deploy-preview-862--windup-documentation.netlify.app/docs/release-notes/master/#rn-known-issues-7-0-2_release-notes)
    * [Resolved issues](https://deploy-preview-862--windup-documentation.netlify.app/docs/release-notes/master/#mta-rn-resolved-issues-7-0-2_release-notes)

### CODE REVIEW

File: docs/topics/mta-rn-resolved-issues-7-0-2.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+:_content-type: REFERENCE
+[id="mta-rn-resolved-issues-7-0-2_{context}"]
+= Resolved issues
+
+The following highlighted issues have been resolved in {ProductFullName} version 7.0.2.
+
+.MTA CLI does not function as expected on Arm-based machines
+
+In previous versions of {ProductShortName} 7.0, there was an issue with the Command-line interface (CLI) not functioning as expected on Arm CPU architecture. This issue has been resolved in {ProductShortName} 7.0.2. link:https://issues.redhat.com/browse/MTA-2160[(MTA-21600)]
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12429435[Resolved Issues in Jira].
```
Comment (anarnold97) [inline docs/topics/mta-rn-resolved-issues-7-0-2.adoc:14]:
```suggestion

For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12429435[Resolved Issues in Jira].
```

File: docs/topics/mta-rn-resolved-issues-7-0-2.adoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+//
+// * docs/release_notes-7.0/master.adoc
+
+:_content-type: REFERENCE
+[id="mta-rn-resolved-issues-7-0-2_{context}"]
+= Resolved issues
+
+The following highlighted issues have been resolved in {ProductFullName} version 7.0.2.
+
+.MTA CLI does not function as expected on Arm-based machines
```
Comment (anarnold97) [inline docs/topics/mta-rn-resolved-issues-7-0-2.adoc:11]:
```suggestion
```suggestion
.CVE-2022-1962: golang: go/parser stack exhaustion in all Parse* functions  Security

A flaw was found in versions of the golang standard library, go/parser before Go 1.17.12 and Go 1.18.4. When calling any Parse functions on the Go source code, which contains deeply nested types or declarations, a panic can occur due to stack exhaustion. This issue allows an attacker to impact system availability.

For more details, see link:https://access.redhat.com/security/cve/CVE-2022-1962[(CVE-2022-1962)].


.MTA CLI does not function as expected on Arm-based machines
```

File: docs/topics/mta-rn-resolved-issues-7-0-2.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+A flaw was found in versions of the golang standard library, go/parser before Go 1.17.12 and Go 1.18.4. When calling any Parse functions on the Go source code, which contains deeply nested types or declarations, a panic can occur due to stack exhaustion. This issue allows an attacker to impact system availability.
+
+For more details, see link:https://access.redhat.com/security/cve/CVE-2022-1962[(CVE-2022-1962)].
+
+
+.MTA CLI does not function as expected on Arm-based machines
+
+In previous versions of {ProductShortName} 7.0, there was an issue with the Command-line interface (CLI) not functioning as expected on Arm CPU architecture. This issue has been resolved in {ProductShortName} 7.0.2. link:https://issues.redhat.com/browse/MTA-2160[(MTA-21600)]
+
```
Comment (anarnold97) [inline docs/topics/mta-rn-resolved-issues-7-0-2.adoc:22]:
```suggestion

.Missing text below graphs on Reports page

In previous versions of (ProductShortName} {DocInfoProductNumber, the text below graphs was missing on *Reports* page. This has been resolved in (ProductShortName} 7.0.2. link:https://issues.redhat.com/browse/MTA-1868[(MTA-1868)]
```

File: docs/topics/mta-rn-resolved-issues-7-0-2.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+.Missing text below graphs on Reports page
+
+In previous versions of (ProductShortName} {DocInfoProductNumber, the text below graphs was missing on *Reports* page. This has been resolved in (ProductShortName} 7.0.2. link:https://issues.redhat.com/browse/MTA-1868[(MTA-1868)]
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12429435[Resolved Issues in Jira].
```
Comment (anarnold97) [inline docs/topics/mta-rn-resolved-issues-7-0-2.adoc:26]:
```suggestion

For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12429435[Resolved Issues in Jira].
```

File: docs/release-notes/master.adoc [hunk 5]
```diff
@@ -17,6 +17,17 @@ include::topics/making-open-source-more-inclusive.adoc[]
 == Introduction
 {ProductName} {DocInfoProductNumber} accelerates large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user interface.
 
+
+[id="mta-7-0-2"]
+== {ProductShortName} 7-0-2
+
```
Comment (anarnold97) [inline docs/release-notes/master.adoc:23]:
```suggestion

include::topics/mta-rn-new-features-7-0-2.adoc[leveloffset=+2]

```

File: docs/topics/mta-rn-known-issues-7-0-2.adoc [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-2_{context}"]
+= Known issues
+
+{ProductFullName} version 7.0.2 has the following known issues.
+
+.Eclipse IDE: {ProductShortName} Run_configuration not functioning as expected
```
Comment (dymurray) [inline docs/topics/mta-rn-known-issues-7-0-2.adoc:11]: Please remove this as the Eclipse IDE will not be published until 7.0.3. So for known issues we can simply mention that the Eclipse IDE is still not available in the marketplace and that customers can still use the 6.2.2 version.

File: docs/topics/mta-rn-new-features-7-0-2.adoc [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+:_content-type: CONCEPT
+[id="rn-new-features-7-0-2_{context}"]
+= New features
+
+
+This section describes the new features and improvements of the {ProductFullName} 7.0.2.
+
+
+.Multiarch support introduced
```
Comment (dymurray) [inline docs/topics/mta-rn-new-features-7-0-2.adoc:13]: Please specify this is for the CLI only

File: docs/topics/mta-rn-known-issues-7-0-2.adoc [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+In {ProductShortName} 7.0.2, the {ProductShortName} *Run_configuration*, within Eclipse, is not functioning as expected. This issue is scheduled to be resolved in {ProductShortName} 7.0.3. link:https://issues.redhat.com/browse/MTA-2264[(MTA-2264)]
```
Comment (anarnold97) [inline docs/topics/mta-rn-known-issues-7-0-2.adoc:13]:
```suggestion
```

File: docs/topics/mta-rn-new-features-7-0-2.adoc (hunk 7 above)
Comment (anarnold97) [inline docs/topics/mta-rn-new-features-7-0-2.adoc:13]:
```suggestion


.Multiarch support for {ProductShortName} CLI introduced
```

File: docs/topics/mta-rn-new-features-7-0-2.adoc [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+[id="rn-new-features-7-0-2_{context}"]
+= New features
+
+
+This section describes the new features and improvements of the {ProductFullName} 7.0.2.
+
+
+.Multiarch support for {ProductShortName} CLI introduced
+
+In {ProductShortName} 7.0.2, multiarch support is introduced. Multiarch support includes both ARM architecture on Linux machines and Apple Silicon Macs.
```
Comment (anarnold97) [inline docs/topics/mta-rn-new-features-7-0-2.adoc:15]:
```suggestion
In {ProductShortName} 7.0.2, multiarch support for for {ProductShortName} CLI is introduced. Multiarch support includes both ARM architecture on Linux machines and Apple Silicon Macs.
```

File: docs/topics/mta-rn-new-features-7-0-2.adoc [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+This section describes the new features and improvements of the {ProductFullName} 7.0.2.
+
+
+.Multiarch support for {ProductShortName} CLI introduced
+
+In {ProductShortName} 7.0.2, multiarch support for for {ProductShortName} CLI is introduced. Multiarch support includes both ARM architecture on Linux machines and Apple Silicon Macs.
+
+
+
+
```
Comment (anarnold97) [inline docs/topics/mta-rn-new-features-7-0-2.adoc:19]:
```suggestion
```

**Comment by anarnold97** [comment]
@dymurray  changes made, you happy? thanks

@sshveta please can I get a review from you? thanks

**Review by sshveta — APPROVED** [review]

File: docs/topics/mta-rn-resolved-issues-7-0-2.adoc [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+
+
+.CVE-2022-1962: Golang: go/parser stack exhaustion in all Parse* functions  Security
+
+A flaw was found in versions of the Golang standard library `go/parser`, before Go 1.17.12 and Go 1.18.4. When calling any Parse functions on the Go source code, which contains deeply nested types or declarations, a panic can occur due to stack exhaustion. This issue allows an attacker to impact system availability.
+
+For more details, see link:https://access.redhat.com/security/cve/CVE-2022-1962[(CVE-2022-1962)].
+
+
+.MTA CLI does not function as expected on ARM-based machines
```
Comment (anarnold97) [inline docs/topics/mta-rn-resolved-issues-7-0-2.adoc:19]:
```suggestion
.{ProductShortName} CLI does not function as expected on ARM-based machines
```

**Pull request merged**